### PR TITLE
Fix get defaultExchange failed when the default exchange not init completed.

### PR DESCRIPTION
**Pull request by zhanghaou**
### Motivation

In some unit tests, default exchange not init completed when call getExchange. 

### Modifications

Add CountDownLatch in ExchangeContainer.java to take control of default exchange init and get.

### Verifying this change
Pass the exist unit tests.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ExchangeContainer.java [hunk 1]
```diff
@@ -27,24 +29,20 @@
...
-
 /**
  * Container for all exchanges in the broker.
  */
 @Slf4j
 public class ExchangeContainer {
 
     private static final Map<String, AmqpExchange.Type> BUILDIN_EXCHANGE_NAME_SET = new HashMap<>();
-    private static PulsarService pulsarService;
+    private static Map<NamespaceName, CountDownLatch> countDownLatchMap = new HashMap<>();
```
Comment (codelipenghui) [inline amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ExchangeContainer.java:39]: One countdown latch is enough, we don't need to introduce a map here.

Comment (zhanghaou) [reply amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ExchangeContainer.java:39]: Each Vhost has its own default exchanges, only one countdownLatch can not work well in the condition of creating multiple Vhost at the same time.

Comment (codelipenghui) [reply amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ExchangeContainer.java:39]: Is the countdown latch only used when the broker startup? or namespace bundle loaded? If just the broker startup, I think you can get how many vhosts. If on the namespace bundle loaded or client trigger load default exchanges, we should avoid introducing a block operation here.

Comment (zhanghaou) [reply amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ExchangeContainer.java:39]: It is used on the namespace bundle loaded, if don't add this, should we need to handle this problem？

**Comment by zhanghaou** [comment]
It will impl in new PR.

**Pull request closed without merging**